examples/gameserver.yaml

```diff
@@ -86,6 +86,23 @@ spec:
   #   # set this GameServer's initial player capacity
   #   initialCapacity: 10
   #
+  # [Stage:Alpha]
+  # [FeatureFlag:CountsAndLists]
+  # Lists provides the configuration for generic (player, room, session, etc.) tracking features.
+  # Commented out since Alpha, and disabled by default
+  # lists:
```

Comment:
This doesn't look right -- should be separate `counts` and `lists`.

Reply:
I only wrote a sample for the lists here and not a sample for couters. I can add that in.

Reply:
This also doesn't match the design. Lists don't have a count value either on them in the spec.